gradle.properties

```diff
@@ -22,7 +22,7 @@ mockitoVersion=5.3.1
 jacocoVersion=0.8.10
 wso2OrbitAntlrVersion=4.5.1.wso2v1
 ballerinaGradlePluginVersion=2.0.1
-bouncycastleVersion=1.74
+bouncycastleVersion=1.78
```

Comment:
Any reason to address bouncy castle vulnerability here? HTTP has not yet addressed this in update 8

Reply:
This is also failing in distribution trivy scan - https://github.com/ballerina-platform/ballerina-distribution/actions/runs/9460736163/job/26060086567

Reply:
Lets proceed with this change, I'll release http to support this change